src/utils/image.cpp

```diff
@@ -18,15 +18,21 @@
 #include <Poco/Process.h>
 #include <Poco/SHA2Engine.h>
 #include <Poco/StreamCopier.h>
+#include <Poco/StringTokenizer.h>
 
 #include "utils/image.hpp"
 
 namespace fs = std::filesystem;
 
 namespace {
+
 const std::unordered_map<std::string, std::regex> cAnchoredEncodedRegexps
     = {{"sha256", std::regex(R"(^[a-f0-9]{64}$)")}, {"sha384", std::regex(R"(^[a-f0-9]{96}$)")},
         {"sha512", std::regex(R"(^[a-f0-9]{128}$)")}};
+constexpr auto cFilePermissionStrLen     = 10;
```

Comment:
Too specific to one particular function could you move this const inside that function?